spec/spec_helper.rb

```diff
@@ -1,3 +1,8 @@
+require 'simplecov'
+SimpleCov.start do
+  add_filter %r{^/spec/}
```

Comment:
Hey @jitingcn what do you think about adding the `minimum_coverage 86.80` option here also?

86.80 come from the current coverage status (I've reduced the value a little bit in order to prevent any flaky test break your PR by not reaching out the minimum coverage):

> Coverage report generated for RSpec to /home/runner/work/meilisearch-rails/meilisearch-rails/coverage. 509 / 586 LOC (86.86%) covered.

https://github.com/simplecov-ruby/simplecov#minimum-coverage

Reply:
Good idea, will add this